apps/frontend/src/components/global/upload_tabs/splunk/AuthStep.vue

```diff
@@ -105,11 +118,7 @@ export default class AuthStep extends Vue {
       localUsername.set(this.username);
       localPassword.set(this.password);
       localHostname.set(this.hostname);
-      if (this.indexToShow === undefined) {
-        localSplunk2HDFIndex.set(this.index);
```

Comment:
i saw that you removed the default of 'hdf' as the index.  is there no need for the splunk2hdf thing anymore?